src/actions/exchangeActions.js

```diff
@@ -153,15 +153,23 @@ export const searchOffersAction = (fromAssetCode: string, toAssetCode: string, f
     // we're requesting although it will start delivering when connection is established
     const { error } = await exchangeService.requestOffers(fromAssetCode, toAssetCode);
 
+    const { isAllowed = false, alpha2 = '' } = await exchangeService.getIPInformation();
+
     api.fetchMoonPayOffers(fromAssetCode, toAssetCode, fromAmount).then((offer) => {
       if (!offer.error) {
-        dispatch({ type: ADD_OFFER, payload: offer });
+        const modoffer = { ...offer, isAllowed, alpha2 };
```

Comment:
How about removing `alpha2` and changing `isAllowed` to `offerRestricted: string` and if it is present the we display whole message without need to pass.

Also add it to Offer model.